layers/+themes/themes-megapack/packages.el

```diff
@@ -37,7 +37,8 @@
     dracula-theme
     espresso-theme
     farmhouse-theme
-    firebelly-theme
+    ;; Removed from melpa
+    ; firebelly-theme
```

Comment:
Sorry that I have to nitpick here, but comments should start with two semicolons (`;;` instead of `;`). Same goes for the rest of the comments.